os/board/common/Kconfig

```diff
@@ -66,6 +66,12 @@ config ARCH_BOARD_HAVE_SECOND_FLASH
 if ARCH_BOARD_HAVE_SECOND_FLASH && (NFILE_DESCRIPTORS != 0)
 comment "Second Flash Partition Options"
 
+config FLASH_SECOND_START_ADDR
+	hex
+	default 0xA000000 if ARCH_BOARD_RTL8730E 
```

Comment:
0xA... is valid for all Rev now? As far as I know, HW Rev. 07 is 0xA... But previously it was 0x9...

Reply:
with the merged linux tool, we are unable to download anything between the region 0x9 to 0xA
https://github.com/Samsung/TizenRT/commit/d4d63dd8748aa9fe610c73a6cc78ef0904e646b7

Reply:
@Tirtha-Samsung PR #6364 also has the same config. Need to talk with @jeongarmy .

Reply:
I think we need to define this value like FLASH_START_ADDR in each board Kconfig.
Please check #6364 changes and give your opinion.